deploy/operator-local.yaml

```diff
@@ -51,6 +51,8 @@ spec:
               value: quay.io/eclipse/che-plugin-registry:7.11.0
             - name: IMAGE_default_devfile_registry
               value: quay.io/eclipse/che-devfile-registry:7.11.0
+            - name: IMAGE_default_che_tls_secrets_creation_job
+              value: quay.io/eclipse/che-tls-secret-creator:alpine-08c8aee
```

Comment:
It is needed automatically update this image on every release
Here is I find the latest ub8_minimal image and update all references
https://github.com/eclipse/che-operator/blob/master/make-release.sh#L27-L30

Reply:
This image is used to generate TLS certificates and create corresponding secrets. It just should work. We don't have to update it on newer releases.